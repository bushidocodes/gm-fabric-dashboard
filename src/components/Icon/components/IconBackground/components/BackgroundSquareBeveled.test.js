import React from "react";
import { shallow } from "enzyme";
import BackgroundSquareBeveled from "./BackgroundSquareBeveled";

describe("BackgroundSquareBeveled", () => {
  it("matches snapshot", () => {
    const aBackgroundSquareBeveled = shallow(<BackgroundSquareBeveled />);
    expect(aBackgroundSquareBeveled).toMatchSnapshot();
  });
});
