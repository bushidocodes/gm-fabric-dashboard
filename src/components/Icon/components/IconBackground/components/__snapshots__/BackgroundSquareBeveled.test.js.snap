// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundSquareBeveled matches snapshot 1`] = `
<g
  className="iconBackground"
  fill="currentColor"
  fillOpacity="1"
  fillRule="evenodd"
  id="SquareBeveled"
>
  <polygon
    id="Rectangle"
    points="4 0 20 0 24 4 24 20 20 24 4 24 0 20 0 4"
  />
</g>
`;
